cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -0,0 +1,95 @@
+# HTTP Cache configuration.
+#
+# See cdo-varnish/README.md for more information on the configuration format.
+class HttpCache
+  STATIC_ASSETS = {
```

Comment:
It's dangerous to strip all cookies based solely on extensions.

There are some assets (e.g. JPG ones) whose content depends on session state, so in general its not safe to blindly cache based on extension.  Brent has already made a VCL config change based on this;  how can we allow similar flexibility in this DSL?

Rather than an extremely complicated config, it would be nice if we had some simple path conventions where static and non-static (i.e not publically cacheable) assets go so that the rules here became simple path matches.